op/fdg_watcher.py

```diff
@@ -110,41 +140,71 @@ def main():
     parser.add_argument("--username", type=str, default="", help="email username")
     parser.add_argument("--password", type=str, default="", help="email password")
     parser.add_argument("--test_email", type=bool, default=False, help="send a test email when start")
```

Comment:
It's recommended to use `action='store_true'` instead, because for `type=bool`, `--test_email False` will be true, which may lead to unexpected result.